src/js/DropContainer.jsx

```diff
@@ -0,0 +1,281 @@
+/*eslint consistent-this: [2, "that"]*/
+
+import React from 'react';
+import classNames from 'classnames';
+import accepts from 'attr-accept';
+import $ from 'jquery';
+
+import css from '../stylesheets/button.css';
+import UploadButton from './UploadButton';
+import ImagePreview from './ImagePreview';
+
+export default class DropContainer extends React.Component {
+   /**
+      * Base Container
+      * @param  {Object} props
+   */
+   constructor(props) {
+      super(props);
+      this.state = {
+         imageFile: '',
```

Comment:
Let's rename this imageFileUrl or something more descriptive. My first thought was that it was referencing the raw File object and not the image preview.